.github/workflows/linter.yml

```
@@ -0,0 +1,48 @@
+###########################
+## Linter GitHub Actions ##
+###########################
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+# For the Web Almanac we have the following rule changes from the defaults:
+#  * Markdown - we're a bit more relaxed about this since authors may be less technical
+#  * Python - Only change from default is to set ignored-modules=flask, flask-talisman...etc.
+#  * CSS - Tweaked to ignore some Prince/Normalise exceptions and some whitespace exceptions
+#  * JS - default rules
+#  * JSON - default rules
+#  * YML - default rules
+# Exception config files are in the .github/linters directory
+#
+name: Lint Code Base
+# Start the job on all push and pull requests #
+on:
+  - push
+  - pull_request
+
+# Set the Job
+jobs:
+  lint:
+    # Name the Job
+    name: Lint Code Base
+    # Skip duplicate job for local brances
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    # Set the agent to run on
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code base
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      # Run Linter against code base #
+      - name: Lint Code Base
```

Thanks for removing unnecessary comment bloat. We can assume that those looking at the code can understand the obvious. I have removed some additional inline comments. Even if you chose not to remove the ones I suggest we should, please delete the trailing ` #` from two comments and correct misspelled `brances`.

```suggestion
name: Lint Code Base
on:
  - push
  - pull_request

jobs:
  lint:
    name: Lint Code Base
    # Skip duplicate job for local branches
    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Code
        uses: actions/checkout@v2

      - name: Lint Code Base
```

---

Left the comments in for the steps but removed the rest

---

When the names of steps are clear enough, we do not need to say the same things in the comments. For example, if the name says, `Checkout Code`, then the comment `# Checkout the code base` does not add any value to it. I think, I will stop here, because I am clearly nitpicking now. ;-)

---

Nah you're right. Removed.